contracts/protocols/mainnet/compound_iii/main.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+import "./interfaces.sol";
+import "./helpers.sol";
+
+/**
+ *@title Compund III Resolver
+ *@dev get user position, user configuration, market configuration.
+ */
+contract CompoundIIIResolver is CompoundIIIHelpers {
+    /**
+     *@dev get position of the user for all collaterals.
+     *@notice get position details of the user in a market including overall position data, collaterals, rewards etc.
+     *@param user Address of the user whose position details are needed.
+     *@param market Address of the market for which the user's position details are needed.
+     *@return UserData Overall position details of the user including balances, nonce, rewards and flags.
+     *@return UserCollateralData Data related to the collateral assets for which user has the position.
+     */
+    function getPositionAll(address user, address market)
```

Comment:
@Richa-iitr It should take `address[] memory market`.

Reply:
Done.